providers/edge/src/airflow/providers/edge/example_dags/integration_test.py

```diff
@@ -30,10 +30,10 @@
 from airflow.exceptions import AirflowNotFoundException
 from airflow.hooks.base import BaseHook
 from airflow.models.dag import DAG
-from airflow.models.param import Param
 from airflow.models.variable import Variable
 from airflow.operators.empty import EmptyOperator
 from airflow.providers.common.compat.standard.operators import PythonOperator
+from airflow.sdk.definitions.param import Param
```

Comment:
Ditto here, I wonder if this should be
```suggestion
from airflow.sdk import Param
```

Reply:
We can do, all tests used directly by users essentially.